h2/src/test/org/h2/test/db/TestBigResult.java

```diff
@@ -289,6 +292,39 @@ private void testSortingAndDistinct4SortingOnly(Statement stat, String sql, int
         assertFalse(rs.next());
     }
 
+    private void testLOB() throws SQLException {
```

Comment:
would be nice to test both LOB and CLOB, they have slightly different paths in some places which have occasionally resulted in bugs

Reply:
OK, I'll add CLOBs here too.